pennylane/ops/qubit/matrix_ops.py

```diff
@@ -296,9 +293,10 @@ def compute_decomposition(U: TensorLike, wires: WiresLike):
                 raise DecompositionUndefinedError(
                     "The decomposition of a two-qubit sparse QubitUnitary is undefined."
                 )
+
             return qml.ops.two_qubit_decomposition(U, Wires(wires))
 
-        return super(QubitUnitary, QubitUnitary).compute_decomposition(U, wires=wires)
+        return qml.ops.op_math.decompositions.multi_qubit_decomposition(U, Wires(wires))
```

Comment:
Why this is changed?

Reply:
This was to show the error that there was no decomposition implemented. now there is always decomposition